JSDemos/Demos/Scheduler/CustomDragAndDrop/Vue/data.ts

```diff
@@ -1,4 +1,8 @@
-export const tasks = [
+export interface Task {
+  text: string;
+}
```

Comment:
why are you using an **interface** and not a **type**? This is not critical, but it seemed to me that we use the type everywhere

Reply:
I just prefer `interface`, there's no particular reason for that
In angular demos there were `interface` too